channeldb/kvdb/etcd/db.go

```diff
@@ -248,18 +253,18 @@ func (db *db) PrintStats() string {
 	return ""
 }
 
-// BeginReadTx opens a database read transaction.
+// BeginReadWriteTx opens a database read+write transaction.
```

Comment:
Weird how the linter just occasionally misses cases like this.

Reply:
yeah